bigtable/metrics.go

```diff
@@ -144,10 +150,17 @@ type builtinMetricsTracerFactory struct {
 	retryCount         metric.Int64Counter
 }
 
-func newBuiltinMetricsTracerFactory(ctx context.Context, project, instance, appProfile string, metricsProvider MetricsProvider) (*builtinMetricsTracerFactory, error) {
+func wrapMetricsError(err error) error {
+	if err == nil {
+		return err
+	}
```

Comment:
nit: Is the `nil` case really that necessary? Most of the time this is being called is because `err` is non-null, and it's also a private function.

Reply:
Yes. This is needed.
In the exporter file, we don't always check whether the err is nil before calling wrap.

E.g. ForceFlush, exportTimeSeries, etc